multiversx_sdk_cli/cli_wallet.py

```diff
@@ -56,7 +63,7 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
     sub.add_argument("--infile", help="path to the input file")
     sub.add_argument("--outfile", help="path to the output file")
     sub.add_argument("--in-format", required=True, choices=WALLET_FORMATS, help="the format of the input file")
-    sub.add_argument("--out-format", required=True, choices=WALLET_FORMATS, help="the format of the output file")
+    sub.add_argument("--out-format", required=True, choices=WALLET_FORMATS_AND_ADDRESSES, help="the format of the output file")
```

Comment:
Now I see that `wallet bech32` can be absorbed into this command in the future, as well. E.g. `in-format=public-key, --out-format=bech32-address`.